languages/java/README.md

```diff
@@ -63,10 +68,19 @@ UUID secretId = secretResponse.getID();
 var secretResponse = bitwardenClient.secrets().get(secretId);
 ```
 
+### Get secrets by ids
+
+```java
+SecretsResponse secretsByIds = bitwardenClient.secrets().getByIds(new UUID[]{secretResponse.getID()});
```

Comment:
```suggestion
SecretsResponse secretsByIds = bitwardenClient.secrets().getByIds(new UUID[]{secretId});
```

Reply:
Updated in https://github.com/bitwarden/sdk/pull/1005/commits/a616e65af26e45037fd444d496aa7d2421b001b8